tests/__helpers__/configuration.ts

```diff
@@ -35,6 +62,25 @@ export const groupConfig = {
   },
 };
 
+export const customGroupConfig = {
+  data: {
+    epic_feature_enabled: true,
+  },
+  staffConfiguration: {
+    epic_feature_enabled: true,
+  },
+  patientConfiguration: {
+    epic_feature_enabled: true,
+  },
+};
+
+export const customGroupConfigResponse = {
+  ...customGroupConfig,
+  id: randomHexString(24),
+  updateTimestamp: '2018-04-24T11:57:44.525Z',
+  creationTimestamp: '2018-04-24T11:57:44.525Z',
+};
+
 export const groupConfigResponse = {
```

Comment:
Should this one be up together with the groupConfig still?

Reply:
I'm not sure I see or understand what you mean? 😅

Reply:
You have put two object between `groupConfig` (line 53)  and `groupConfigResponse` (line 84) while those objects itself belong together.

Reply:
Updated the file to be more clear